src/lib/errors.ts

```diff
@@ -48,3 +55,10 @@ export class NotAllowedError extends HttpErrorHandler {
     Object.setPrototypeOf(this, NotAllowedError.prototype);
```

Comment:
type should be `ChangesetAlreadyClosedError`

Reply:
it should be as it is. the code was hided you had to extend it.